website/views/options/social/group.jade

```diff
@@ -147,11 +147,26 @@ a.pull-right.gem-wallet(ng-if='group.type!="party"', popover-trigger='mouseenter
                 h3.popover-title {{group.leader.profile.name}}
                 .popover-content
                   markdown(text='group._editing ? groupCopy.leaderMessage : group.leaderMessage')
-      div(ng-controller='ChatCtrl')
+                  
+      ul.options-menu(ng-init="groupPane = 'chat'", ng-show="group.purchased.active")
+        li
+          a(ng-click="groupPane = 'chat'")
```

Comment:
I thought about that. But, chat is the default, so that would make the default group url always have chat. Plus, groups with out a subscription would always have the chat url as well. I think maybe in the future it would be a good idea.